### ISSUE TITLE
Navbar should be more apparent

### ISSUE THREAD

**Issue by lucasburlingham** [JovarkOS/docs#1]
Navbar dropdowns should have different background color than the `body` of the page for UI/UX design.

**Comment by Hariomsingh333** [comment]
hey, can I work on this project?

**Comment by Hariomsingh333** [comment]
hey there, i make a new branch and fix the issue can i push

**Comment by lucasburlingham** [comment]
Yes, I would love to add you to the project! If you make a pull request, I will review it.

**Comment by Hariomsingh333** [comment]
ok

**Comment by Hariomsingh333** [comment]
I fix the issue and I hope it should help you, I make the  pull request,

[link](https://github.com/JovarkOS/docs/pull/3)
